features/content_crop.feature

```diff
@@ -270,7 +269,41 @@ Feature: Cropping the Image Articles
         }
       }
       """
-
+      When we get "/archive/bike"
+      Then we get OK response
+      And we get no "poi"
+      When we patch "/archive/feature_image"
+      """
+      {
+        "associations": {
+          "featureimage": {
+            "_id": "bike",
+            "poi": {"x": 0.1, "y": 0.1},
+            "renditions": {
+              "16-9": {"CropLeft":0,"CropRight":1280,"CropTop":0,"CropBottom":720}
```

Comment:
This PR doesn't introduce this feature. And the response bellow is truncated. Only what I want to check appears.
But it's a good remark. How can it be more RESTfull from your point of view ?